_posts/en/2020-10-27-cio-digital-transformation.md

```diff
@@ -0,0 +1,216 @@
+If a client changed his mind during this execution phase, the impact to the project was high.
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/digital_feedback_loop.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/pmlc_current_model.png"
+        alt = "This image depicts the current Project Management Lifecycle. It shows 4 steps (Inception, Planning, Execution, and Closure). Each of the steps is in sequence where, to start the Execution step, we are expected to finish the Planning step. At each step, there is an increasing amount of documentation being produced and stakeholders involved. Culminating to the Execution step where IT personnel are to execute and make the software changes. The client is shown at the beginning and at the end of the process, but not in the middle of it."
+        style="width: 700px; height: inherit;"
+        />
+    </a>
+    <small>Figure 2. A high-level view and interpretation of the current Project Management Lifecycle using gated decision points</small>
+</div>
+
+The bigger the project, the further into the future we require to foresee.
+This approach comes with a few challenges:
+
+1. ***Accurate foresight is extremely rare***: There are statistics and reports regarding the low success rates of large projects[^2].
+Though we think those are not necessarily about success rates.
+They are about the reality that change is inevitable, that having such foresight is extremely rare, and that it is nearly impossible to plan that far into the future.
+
+2. ***A business case is thought of as an expectation, instead of a hypothesis***: The benefits realization at the end of a project is not a sure thing.
+A project may be delivered on time, on budget, as per requirements, but [yield negative outcomes](https://sencanada.ca/content/sen/committee/421/NFFN/reports/NFFN_Phoenix_Report_32_WEB_e.pdf).
+The longer the organization is locked in a project, the greater the risk of being locked into a potentially bad idea.
+
+3. ***It puts IT in a passive state***: "_I need requirements_" is something you probably heard your IT teams say to you.
+This should not be surprising as we are framing the organization to behave as such.
+The [Government's pandemic response](https://nationalpost.com/opinion/john-ivison-amid-staggering-unemployment-rate-public-servants-processing-ei-claims-are-the-unsung-heroes) is an exception to this statement where personnel worked creatively together in a time of crisis.
+
+4. ***Progression is measured with documents, instead of working software***: If after 18 months of work and $2M spent we do not have working software to show for, would we consider it a good investment?
+
+> Only **successful projects, in production**, enable the organization to gain **empirical evidence** necessary for their **evidence-based decisions**.
+
+## Opportunities for change
```

Comment:
Should we acknowledge challenges to the proposed approach?

Taking this road needs senior leaders to acknowledge the benefits and take key actions to tackle challenges:
Change of governance, HR strategy to acquire knowledge while maintaining existing procedures, new type of architecture which challenge the current principles in place when we acquire technology, current data management practice, etc.